src/Meilisearch/Meilisearch.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>netstandard2.0</TargetFramework>
+        <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
@juchom do you think is good to tell people which .net version this package supports in the README - being explicitly about that?

Reply:
Good question, I went to a couple of repositories and this is not specified. I don't have any opinion on this.

If you want to this to the README, just give me the text and I'll update the issue.

Reply:
Awesome @juchom 💯 

You could add in line 48 of `README.md`.

`This package requires .NET Core 3.1 or later.`